pkg/session/bootstrap_test.go

```diff
@@ -2219,11 +2219,17 @@ func TestTiDBUpgradeToVer209(t *testing.T) {
 }
 
 func TestTiDBUpgradeWithDistTaskEnable(t *testing.T) {
+	if kerneltype.IsNextGen() {
+		t.Skip("next-gen kernel system tables have different clustered pk setting, dropping pk will fail in upgradeToVer248")
```

Comment:
```suggestion
		t.Skip("the schema version of the first next-gen kernel release is 250, no need to go through upgrade operations below it, skip it")
```
please modify other messages too